style.css

```diff
@@ -0,0 +1,59 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 20px;
+}
+
+.hamburger-menu {
+  border-color: transparent;
+}
+
+.hamburger-menu:hover {
+  cursor: pointer;
+}
+
+.hamburger-menu:active {
+  cursor: context-menu;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  background-image: url('./img/header-background-mobile.png');
+  background-repeat: no-repeat;
+  font-style: normal;
+  margin: 10px 0;
+  padding: 95px 20px;
+}
+
+h1 {
+  width: 250px;
+  height: 95px;
+  font-weight: 700;
+  font-size: 40px;
+}
+
+.lets-connect {
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
+}
+
+.headline-text {
+  width: 327px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
```

Comment:
As requested by the previous reviewer, please make sure to update your introduction section to look as similar to the template as possible 😉. Currently, there are some small differences:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/205210578-7b095385-1dd2-4478-9f1a-45336fc3c779.png)  | ![image](https://user-images.githubusercontent.com/42799579/205210599-ef3696f0-32ce-48fc-80db-2163afe43de2.png)  |

Points to notice:
- The hamburger icon in the original design has a transparent background.
- Please be sure to use the correct fonts displayed on the design 😉. I will explain more in the comment below.
- The text "LET'S CONNECT" should be bold.
- Please add all media icons.